docs/core/tools/global-tools.md

```diff
@@ -0,0 +1,166 @@
+---
+title: .NET Core Global Tools
```

Comment:
Question about naming (for @livarcocc @KathleenDollard). At one point, I remember us saying we wanted to call this feature ".NET Core tools" because 'global' is just one way to install the tool, but not the only one. Am I remembering right? Has the thinking changed? Or have we settled on calling this feature ".NET Core Global Tools"?